Add Terraform v1.13 Schematics Support

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Description

IBM [recently added](https://cloud.ibm.com/status/announcement?component=schematics) Terraform v1.13 support for Schematics with v1.12 being the new default since December 8, 2025. While upgrading to a newer version, I noticed that the provider does not yet support versions more recent than v1.10.
The logs report errors like these:
```
[ERROR] vertex "ibm_schematics_workspace.workspace (expand)" error: "template_type" ("terraform_v1.13") should match regexp ^terraform_v(?:1\.4|1\.5|1\.6|1\.7|1\.8|1\.9|1\.10)(?:\.\d+)?$
```

That regex is located [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/6b224c5d5eabba6306179c03591b8727ff54b0f1/ibm/service/schematics/resource_ibm_schematics_workspace.go#L584). I'm not sure if modifying it to permit the new versions is enough or if other changes are required.

### New or Affected Resource(s)

<!--- Please list the new or affected resources and data sources. --->

* ibm_schematics_workspace

--- a/ibm/service/schematics/resource_ibm_schematics_workspace.go
+++ b/ibm/service/schematics/resource_ibm_schematics_workspace.go
@@ -581,7 +581,7 @@ func ResourceIBMSchematicsWorkspaceValidator() *validate.ResourceValidator {
 			Identifier:                 schematicsWorkspaceTemplateType,
 			ValidateFunctionIdentifier: validate.ValidateRegexp,
 			Type:                       validate.TypeString,
-			Regexp:                     `^terraform_v(?:1\.4|1\.5|1\.6|1\.7|1\.8|1\.9|1\.10)(?:\.\d+)?$`,
+			Regexp:                     `^terraform_v(?:1\.4|1\.5|1\.6|1\.7|1\.8|1\.9|1\.10|1\.11|1\.12|1\.13)(?:\.\d+)?$`,
 			Default:                    "[]",
 			Optional:                   true})
 

